api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -519,6 +531,14 @@
      */
     Builder setAttribute(String key, AttributeValue value);
```

Comment:
Radical suggestion: Remove the plain String overloads? I think using semantic attributes should be the more usual case, so having a slightly more complicated API for using "raw" strings could be acceptable.

Reply:
hmmm. It would certainly add significant pain for manual instrumentation that had nothing to do with semantic conventions, though.

Reply:
If we renamed the create method of the key ckasses to something that can be used with `import static` it would be a bit better.

Reply:
do you have some pseudo code? I'm don't really understand how you would do this.

Reply:
Right now we have e.g. `public static DoubleAttributeSetter create(String attributeKey)` . If we renamed that to e.g. `doubleKey`, you could do, in your manual instrumentation code:

```java
import static io.opentelemetry.trace.attributes.DoubleAttributeSetter.doubleKey;
// ...

mySpan.setAttribute(doubleKey("foo"), 4.2);
```

Reply:
Of course, right now you could also do: `private static final DoubleAttributeSetter fooKey = DoubleAttributeSetter.create("foo");` in your instrumentation, which is also a nice place to attach javadoc.

Reply:
changing the name of the static creator method is a very simple change, yes. I'll create a branch where I get rid of the String keys and push it up as a separate draft.

Reply:
See #1097 for removing the bare attribute keys.